src/lib/minject/InjectorMacro.hx

```diff
@@ -230,6 +219,17 @@ class InjectorMacro
 				switch (field.type)
 				{
 					case TFun(args, _):
+						if (post != null)
```

Comment:
Rather than have the order in a temporary-to-be-discarded object at compile time, I've added it as the 2nd piece of metadata on function calls (after the field name, before the argument info).

If it's a blank string, then it's a MethodInjectionPoint or ConstructorInjectionPoint.  If it's not, then the value is the post order.

I've done it as a String to keep with the `rtti:Array<String>` type.  You could conceivably use an integer instead though and have `rtti:Array<Dynamic>`.  Up to you.